Makefile

```diff
@@ -101,10 +101,10 @@ test-all: test-local test-sagemaker
 
 # Builds and installs sagemaker-python-sdk-spark library, since it's used in sagemaker tests.
 install-sdk:
-	cd sagemaker-python-sdk-spark; make
+	pip install --upgrade sagemaker>=2.9.0
 
 clean-sdk:
-	cd sagemaker-python-sdk-spark; make clean
+	rm =*
```

Comment:
I've never seen `=*`, what does it do?

Reply:
so somehow pip install sagemaker generates a file named "=2.9.0" so this is just to remove that file

Reply:
I tried pip install sagemaker, but didn't get a file named "=2.9.0" -- could we remove this target instead? Seems like we can if we're removing this whole directory

Reply:
Removed